Deploy/Build-Module.ps1

```diff
@@ -16,7 +16,7 @@ If (-not $ADMUGetConfig) {
 }
 ###########################################################################
 # Region Checking PowerShell Gallery module version
-Write-Host ('[status]Check PowerShell Gallery for module version info')
+Write-Host ('[status] Check PowerShell Gallery for module version info')
```

Comment:
Only reason for this change is tech debt, I wanted to see all these status messages appear similar. Some had spaces, others didn't.